typescript/packages/well-log-viewer/src/SyncLogViewer.tsx

```diff
@@ -481,8 +501,8 @@ class SyncLogViewer extends Component<SyncLogViewerProps, State> {
     getDefaultPrimaryAxis(): string {
         if (this.props.primaryAxis) return this.props.primaryAxis;
 
-        const _axes = this.props.welllogs?.map((welllog: WellLog) =>
-            getAvailableAxes(welllog, this.props.axisMnemos)
+        const _axes = this.wellLogCollections?.map((collection) =>
```

Comment:
fallback on welllogs ?

Reply:
`this.wellLogCollections` is the internal value that already has applied the fallback (see :334)
```ts
this.wellLogCollections = getWellLogCollectionsFromProps(props);
```

Reply:
OK. Thanks